### ISSUE TITLE
Duplicate Albums and Title in file name when processed multiple times

### ISSUE THREAD

**Issue by jalsin82** [jmathai/elodie#172]
If a file is imported multiple times, the title and/or album is continually added to the filename. Is it possible to prevent this duplication?

**Comment by jmathai** [comment]
Labeled this a bug. Should be able to fix it.

**Comment by jmathai** [comment]
I'm not able to reproduce this. I imported a file and then imported it again. The final name of the file the second time is `2015-12-05_08-59-26-with-title-some-title.jpg` and only includes the title once.

Can you provide steps to reproduce?

```
./elodie.py import --destination=/Users/jaisen/Downloads/test  /Users/jaisen/Documents/2015-12-05_08-59-26-with-title-some-title.jpg  --allow-duplicates
/Users/jaisen/Documents/2015-12-05_08-59-26-with-title-some-title.jpg -> /Users/jaisen/Downloads/test/2015-12-Dec/Unknown Location/2015-12-05_08-59-26-with-title-some-title.jpg
```

**Comment by jalsin82** [comment]
Unfortunately, I don’t have time anymore to play with this.

All I can remember from my testing was having the program process pictures and file them in folders, then moving them back to an “original” folder and processing them again. I did this many times, using many different options, All I know is that I identified that, if I gave the picture a title and/or an album name, that text continued to be duplicated each time the picture was processed.

I’m sorry this response was delayed; I was hoping to be able to play with it more and give you better details. I’ve determined that I will not have the ability to do this for quite some time.

--
Jason Alsin
jcalsin@hotmail.com<mailto:jcalsin@hotmail.com>

**Comment by ldexterldesign** [comment]
Hey,

I'm experiencing this issue right now

Before: ``2013-01-24_20-13-33-imag0055.jpg``
After: ``2013-01-24_20-13-33-2013-01-24_20-13-33-imag0055``

Let me know what info' you need and hopefully we can resolve it

Hope this is useful

Yours hopefully

**Comment by jmathai** [comment]
This is a separate issue. I'll look into it more but there was a change where we add the original file name to the EXIF and rely on that. This command will modify all of your files by adding the original file name to the EXIF so feel free to make a backup before running it if you need.

```
python -m elodie.tools.add_original_name /path/to/photo/folder
```

**Comment by ldexterldesign** [comment]
I'll leave it for now (month/location folders enough organisation for me ATM), but https://github.com/jmathai/elodie/issues/107 soon come!

I will clean up all my files with random (alpha)numeric string:

[yyyy]-[mm]-[dd]_[hh]-[mm]-[ss]-[random]

Do you envision this will be possible in the future?

Kind thanks
